interactive/api/static/examples/CHAPMAN/camp_data/species.json

```diff
@@ -3,8 +3,8 @@
     {
       "name": "M",
       "type": "CHEM_SPEC",
-      "tracer type": "CONSTANT",
-      "description": "Third-body molecule. This is any molecule present in the system."
+      "tracer type": "THIRD_BODY",
+	  "__description": "Third-body molecule. This is any molecule present in the system."
```

Comment:
Sorry, I don't want to be obsessed with the formatting but can we use the uniform indentation? If your tab key has the default spacing that is larger than the given format, I think you could use space to match the same indentation

Reply:
How many spaces is the default spacing?

Reply:
Looks like 2